run_continuous.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+argument="--continuous"
+./run.sh "$argument"
```

Comment:
If you pass “$@“ here then —help and other arguments will work for the run-continuous.sh script.

Reply:
He just copies the original silliness in the Windows script lol, it's ok. The entire argument line is not necessary.

Reply:
> If you pass “$@“ here then —help and other arguments will work for the run_continuous.sh script. Users might think that they passed arguments but those will be not sent to run.sh eventually to autogpt which might cause confusion.

@hdkiller & @nponeccop 

Thanks for the feedback, I appreciate it.
So basically I should change it to this: 
```
#!/bin/bash

./run.sh --continuous "$@"
```
This way the `--continuous` flag will be added to the command-line arguments, and any other arguments provided by the user will be passed through to the run.sh script, and eventually to the autogpt module.

Making the change now

Reply:
Should be --continuous $@ without double quotes, I guess?

Reply:
> Should be --continuous $@ without double quotes, I guess?

The quotes around "$@" are to preserve the original command-line arguments, including any arguments that contain spaces or other special characters. 

For instance, if you call `myscript.sh` with the following arguments:
```
./myscript.sh "arg1 with spaces" arg2 "arg3 with spaces"
```

Using "$@" will pass the arguments to `another_script.sh` exactly as they were received, preserving the spaces:

- arg1 with spaces
- arg2
- arg3 with spaces

If you were to use $@ without the quotes, the arguments would be split by spaces, and `another_script.sh` would receive them as separate arguments:

- arg1
- with
- spaces
- arg2
- arg3
- with
- spaces

Reply:
This is the intended behavior, isn't it? We just forward the arguments as is. If our original script received 15 arguments, the nested script receives 15, and not one big argument with spaces inside.

Reply:
> This is the intended behavior, isn't it? We just forward the arguments as is. If our original script received 15 arguments, the nested script receives 15, and not one big argument with spaces inside.

Ok, adjusting it now

Reply:
> This is the intended behavior, isn't it? We just forward the arguments as is. If our original script received 15 arguments, the nested script receives 15, and not one big argument with spaces inside.

done